tools/diagnostic-information

```diff
@@ -131,6 +137,19 @@ system_logs() {
     cp -H -f /var/log/tntnet/www-audit* "${WD}/${TMPDIR}/logs/"
 }
 
+ipm_logs() {
+    verbose "ipm logs..."
+    # FIXME: check for activemq logs
+    if [ -d "$KARAF_LOG" ]
```

Comment:
For the case there is no `etn-envvars.env` (e.g. its service not enabled), check also for not-emptiness of `$KARAF_LOG` variable expansion.

On similar note, pack also nginx logs?

Reply:
well, I asked the same to myself for -d Vs -z, and the fact is that empty KARAF_LOG results in the same for the -d test. Still, fixed, thx

Reply:
resloved?... commit not pushed up yet? :)